runtime/lua/vim/_system.lua

```diff
@@ -1,14 +1,14 @@
 local uv = vim.uv
 
 --- @class SystemOpts
---- @field cmd string[]
---- @field stdin string|string[]|true
---- @field stdout fun(err:string, data: string)|false
---- @field stderr fun(err:string, data: string)|false
+--- @field cmd? string[]
```

Comment:
I think this one should be removed. `cmd` is not the first argument to `vim.system`.

Reply:
You mean remove the whole line?

Reply:
yeah